src/styles/index.ts

```diff
@@ -5302,6 +5302,20 @@ const styles = (theme: ThemeColors) =>
             width: variables.sideBarWidth - 19,
         },
 
+        progressBarWrapper: {
+            height: 2,
+            width: '100%',
+            backgroundColor: theme.border,
+            borderRadius: 5,
```

Comment:
NAB but why 5? If the height is only 2 total, couldn't the border radius just be 2 as well?